ui/components/app/collectible-details/index.scss

```diff
@@ -53,18 +53,41 @@ $spacer-break-small: 16px;
     overflow-wrap: break-word;
   }
 
-  &__contract-link,
-  &__image-link {
-    word-break: break-all;
+  &__contract-wrapper {
+    max-width: calc(100% - #{$link-title-width});
+  }
+
+  &__contract-copy-button {
+    @include H6;
 
-    @media screen and (max-width: $break-small) {
-      overflow: hidden;
-      text-overflow: ellipsis;
-      white-space: nowrap;
-      width: 90%;
+    width: 80px;
+    display: flex;
+    align-items: flex-start;
+    justify-content: center;
+    background-color: transparent;
+    cursor: pointer;
+    color: var(--ui-4);
+    border: 0;
+
+    &:active {
+      transform: scale(0.97);
     }
   }
 
+  &__contract-link {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    max-width: 90%;
+  }
+
+  &__image-link {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    max-width: 40%;
```

Comment:
Same here, I'll leave it as there may have been some requirement to align the links. They may appear aligned but they aren't really ;)

Current with `40%` width will always change the alignment depending on the width of the parent container.

<img width="504" alt="Screen Shot 2022-01-26 at 12 29 19 PM" src="https://user-images.githubusercontent.com/8112138/151077392-fb09f043-3e2e-4e6b-a68d-9d84c631b602.png">


Removing the `40%` width aligns it with other elements on the page

<img width="425" alt="Screen Shot 2022-01-26 at 12 30 08 PM" src="https://user-images.githubusercontent.com/8112138/151077399-a5022f65-b527-4558-80e0-d14dd0421754.png">

Reply:
Just re-read the PR description. I'll remove this as it was a design requirement :)

Reply:
This max width should be wrapped in a media query for non-pop up screen sizes. Yeah exactly its to address the design ask.

Reply:
@georgewrmarshall How would you recommend I truncate the link for this design ask?